src/identifiers/models.py

```diff
@@ -29,6 +29,8 @@
     ('doi', 'DOI'),
     ('uri', 'URI Path'),
     ('pubid', 'Publisher ID'),
+    ('arxiv', 'ArXiv ID'),  # https://arxiv.org/
+    ('preprintid', 'Preprint ID'),
```

Comment:
The `'preprintid'` choice is something that is useful when migrating papers from other systems, however please consider it a "weak desiderata" :slightly_smiling_face: